src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -98,6 +113,27 @@ def __init__(
         self.src_path = src_path
 
 
+@dataclass
+class BundleContext:
+    package_name: str
+    artifacts: List[PathMapping]
+    project_root: Path
+    bundle_root: Path
+    deploy_root: Path
+    generated_root: Path
+
+    @staticmethod
+    def from_na_project(na: NativeAppProjectModel) -> BundleContext:
```

Comment:
Given that I assume NAPM has to be removed entirely for v2 eventually, would it make sense to extract a bundlecontext from the current NAPM instead duplicating the data as it's happening in this PR?

Reply:
Yeah, I think you're right. I'll move this method to `NativeAppProjectModel.get_bundle_context()`.